what4/src/What4/Serialize/Log.hs

```diff
@@ -0,0 +1,450 @@
+{-# OPTIONS_GHC -fno-warn-name-shadowing #-}
```

Comment:
Is it too difficult to just remove this line and fix the name shadowing warnings that arise?

Reply:
Fixed